compiler/crates/relay-codegen/src/ast.rs

```diff
@@ -69,6 +69,7 @@ pub enum Primitive {
     Int(i64),
     Bool(bool),
     Null,
+    Elide,
```

Comment:
Oh, love this idea!

This is super minor nit: I wonder if there is a different name for this: `MaybeNull`, `NullOrNothing`, `OptionalNull`? 

As an ESL person, I have to look up this word, and still not entirely sure what it means in my language, but that's me. Great word!